src/earn/EarnPoolInfoScreen.tsx

```diff
@@ -36,6 +44,21 @@ import { NetworkId } from 'src/transactions/types'
 import { navigateToURI } from 'src/utils/linking'
 import { formattedDuration } from 'src/utils/time'
 
+export enum BeforeDepositActionName {
+  Add = 'Add',
+  Transfer = 'Transfer',
+  SwapAndDeposit = 'SwapAndDeposit',
+  CrossChainSwap = 'CrossChainSwap',
+}
+
+export interface BeforeDepositAction {
+  name: BeforeDepositActionName
+  title: string
+  details: string
+  iconComponent: React.MemoExoticComponent<({ color }: { color: Colors }) => JSX.Element>
+  onPress: () => void
+}
```

Comment:
Looks like this causes cyclic imports. Consider moving these to a shared location. E.g., `earn/types`